backend/validators/llm.py

```
@@ -0,0 +1,141 @@
+        self._config.write_default()
+
+    async def terminate(self):
+        if self._terminated:
+            return
+        self._terminated = True
+        await self.stop()
+        self._config.terminate()
+
+    def __del__(self):
+        if not self._terminated:
+            raise Exception("service was not terminated")
```

Pylint (3.3.7)

[warning] Raising too general exception: Exception

(W0719)